ztf_viewer/pages/lc_csv.py

```diff
@@ -18,10 +19,24 @@ def get_csv(dr, oids, min_mjd=None, max_mjd=None):
         oid_df = pd.DataFrame.from_records(lc)
         oid_df["oid"] = oid
         oid_df["filter"] = meta["filter"]
+
+        try:
+            ref = ztf_ref.get(oid, dr)
+        except NotFound:
+            pass
+        else:
+            ref_mag = ref["mag"] + ref["magzp"]
+            ref_err = ref["sigmag"]
+            oid_df["ref"] = ref_mag
+            oid_df["ref_err"] = ref_err
+
         dfs.append(oid_df)
     df = pd.concat(dfs, axis="index")
     df.sort_values(by="mjd", inplace=True)
-    df = df[["oid", "filter", "mjd", "mag", "magerr", "clrcoeff"]]
+    try:
+        df = df[["oid", "filter", "mjd", "mag", "magerr", "clrcoeff", "ref", "ref_err"]]
+    except KeyError:
+        df = df[["oid", "filter", "mjd", "mag", "magerr", "clrcoeff"]]
```

Comment:
If we still want to make it conditional (see the comment above) I would prefer more explicit check here
```suggestion
    if "ref" in df.columns:
        df = df[["oid", "filter", "mjd", "mag", "magerr", "clrcoeff", "ref", "ref_err"]]
    else:
        df = df[["oid", "filter", "mjd", "mag", "magerr", "clrcoeff"]]
```